### PR TITLE
[semver:minor] Fixed  --dry-run flag

### PR DESCRIPTION
**Pull request by brivu**
- Changed --dry-run flag from boolean to enum type
- Removed all tests for Helm version 2.  Updated tests to helm version 3
- Updated  Helm Orb to version 1.2
- Updated Kubernetes Orb to version 1.0

### CODE REVIEW

File: src/commands/install-eksctl.yml [hunk 1]
```diff
@@ -6,18 +6,28 @@ steps:
...
-        mkdir -p eksctl_download
-        curl --silent --location --retry 5 "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_$(uname -s)_amd64.tar.gz" \
-          | tar xz -C eksctl_download
-        chmod +x eksctl_download/eksctl
-        SUDO=""
-        if [ $(id -u) -ne 0 ] && which sudo > /dev/null ; then
-          SUDO="sudo"
+        if ! command -v eksctl >/dev/null 2>&1; then
+          echo "Installing the AWS eksctl CLI"
+            curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
```
Comment (Jaryt) [inline src/commands/install-eksctl.yml:25]: I see we define the current platform in 9-20, but we still use uname here. Is this correct?

Comment (brivu) [reply src/commands/install-eksctl.yml]: fixed in next commit

**Review by Jaryt — APPROVED** [review]
lgtm!

File: .circleci/config.yml [hunk 2]
```diff
@@ -13,17 +13,17 @@ orb_promotion_filters: &orb_promotion_filters
     only: /^(major|minor|patch)-release-v\d+\.\d+\.\d+$/
 
 orbs:
-  aws-eks: circleci/aws-eks@dev:alpha
+  aws-eks: circleci/aws-eks@dev:hotfix
```
Comment (KyleTryon) [inline .circleci/config.yml:16]: Just commenting here to document our intent. This hotfix will be overwritten in the next PR #46

**Pull request merged**